linera-service/src/project.rs

```diff
@@ -276,33 +276,18 @@ impl Project {
     }
 
     /// Adds ['linera-sdk'] dependencies in `release` mode.
-    ///
-    /// Includes the contents of `linera-sdk`'s `Cargo.toml` at compile time
-    /// to figure out the latest version.
     #[cfg(not(debug_assertions))]
     fn linera_sdk_dependencies() -> (String, String, String) {
-        let sdk_toml_contents = include_str!("../../linera-sdk/Cargo.toml");
```

Comment:
I kinda prefer the new solution but if you really want to keep the old behavior, symlinks should work (see `linera-execution`)